website/source/docs/guides/agent-encryption.html.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: "docs"
+page_title: "Agent Communication Encryption"
+sidebar_current: "docs-guides-agent-encryption"
+description: |-
+  This guide covers how to encrypt both gossip and RPC communication.
+---
+
+# Agent Communication Encryption
+
+There are two separate systems that need to be configured separately to encrypt communication within the cluster, gossip encryption and TLS. TLS is used to secure the RPC calls between agents. Gossip encryption is secured with a symmetric key, since gossip between nodes is done over UDP. In this guide we will configure both.
+
+To complete the RPC encryption section, you must have [configured agent certificates](https://www.consul.io/docs/guides/creating-certificates.html).
+
+## Gossip Encryption
+
+To enable gossip encryption, you need to use an encryption key when starting the Consul agent. The key can be set via the `encrypt` parameter in the agent configuration file or in a separate file containing the encryption key. The key must be 16-bytes, Base64 encoded. 
+
+You can use the Consul CLI command, [`consul keygen`](/docs/commands/keygen.html), to generate a cryptographically suitable key.
+
+```sh
+$ consul keygen
+cg8StVXbQJ0gPvMd9o7yrg==
+```
+
+### Enable Gossip Encryption: New Cluster
+
+To enable gossip on a new cluster, we will add the encryption key parameter to the
+agent configuration file and then pass the file at startup with the [`-config-dir`](https://www.consul.io/docs/agent/options.html#_config_dir) flag.
+
+```javascript
+{
+  "data_dir": "/opt/consul",
+  "log_level": "INFO",
+  "node_name": "bulldog",
+  "server": true,
+  "encrypt": "JY34uTPZyfUE+6tinMYEVw=="
+}
+```
+
+```sh
+$ consul agent -data-dir=/tmp/consul -config-dir=/etc/consul.d/
+==> WARNING: LAN keyring exists but -encrypt given, using keyring
+==> WARNING: WAN keyring exists but -encrypt given, using keyring
+==> Starting Consul agent...
+==> Starting Consul agent RPC...
+==> Consul agent running!
+         Node name: 'Armons-MacBook-Air.local'
+        Datacenter: 'dc1'
+            Server: false (bootstrap: false)
+       Client Addr: 127.0.0.1 (HTTP: 8500, HTTPS: -1, DNS: 8600, RPC: 8400)
+      Cluster Addr: 10.1.10.12 (LAN: 8301, WAN: 8302)
+    Gossip encrypt: true, RPC-TLS: false, TLS-Incoming: false
+...
+```
+
+"Encrypted: true" will be included in the output, if encryption is properly configured.
```

Comment:
`Encrypt: Gossip: true,...` is what you see, not `Encrypted: true`

Reply:
That was another carry over from the docs. I will update it in both places.